regulations/static/regulations/js/source/collections/comment-collection.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var $ = require('jquery');
+var Backbone = require('backbone');
+Backbone.LocalStorage = require('backbone.localstorage');
+
+var CommentModel = require('../models/comment-model');
+
+var CommentCollection = Backbone.Collection.extend({
+  model: CommentModel,
+  localStorage: new Backbone.LocalStorage('comment:' + window.DOC_NUMBER)
```

Comment:
Thinking about this from a modeling perspective, I'd think that the FR doc number would be an _attribute_ of an element in the collection, not part of the collection itself. I'd expect to filter by doc_number when retrieving, if necessary.